openage/convert/entity_object/export/media_export_request.py

```diff
@@ -149,7 +153,7 @@ class TerrainMediaExportRequest(MediaExportRequest):
     def get_type(self):
         return MediaType.TERRAIN
 
-    def save(self, sourcedir, exportdir, palettes, game_version, *args, **kwargs):
+    def save(self, sourcedir, exportdir, palettes, game_version, compression_level, *args, **kwargs):
```

Comment:
Pylint will complain twice on this line (line too long, too many arguments 6/5).

I didn't find anything in code style that would address long function declaration. Please advice what is your preferred code style.

Is silencing pylint on this particular line ok?

Reply:
You can split the function declaration into two lines here and suppress the pylint output for `too-many-arguments`.